spec/abstractops.html

```diff
@@ -129,6 +129,68 @@ <h1>ToDurationLike ( _duration_ )</h1>
     </emu-table>
   </emu-clause>
 
+  <emu-clause id=sec-temporal-ToTimeDurationLike aoid=ToTimeDurationLike>
+    <h1>ToTimeDurationLike ( _duration_ )</h1>
+    <emu-alg>
+      1. Let _result_ be {
```

Comment:
is this meant to be a Record, or an object?

Reply:
Record.